ansible/configs/sap-smart/sat_workload.yml

```diff
@@ -0,0 +1,122 @@
+    satellite-installer
+      --scenario satellite
+      --foreman-initial-organization "{{ satellite_organization }}"
+      --foreman-initial-location "{{ satellite_location }}"
+      --foreman-initial-admin-username "{{ satellite_username }}"
+      --foreman-initial-admin-password "{{ satellite_password }}"
+      --certs-cname "sat-{{ guid }}.{{ guid }}.{{ osp_cluster_dns_zone }}"
+      --foreman-proxy-dns-managed=false
+      --foreman-proxy-dhcp-managed=false
+
+- name: Prevent Satellite to connect to access.redhat.com
+  lineinfile:
+    path: /etc/hosts
+    line: 127.0.0.1 access.redhat.com
+    create: yes
+
+- name: "Copy subscription manifest .zip file"
+  copy:
+    src: "{{ manifest_file_path }}"
+    dest: /root/manifest.zip
+
+- name: "Add manifest to Satellite"
+  command: >
+    hammer subscription upload
+    --file /root/manifest.zip
+    --organization "{{ satellite_organization }}"
+
+- name: "Enable repositories for specific minor release"
+  command: > 
+    hammer
+      -u "{{ satellite_username }}"
+      -p "{{ satellite_password }}"
+      repository-set enable 
+      --organization "{{ satellite_organization }}"
+      --product "{{ item.product }}"
+      --name "{{ item.name }}"
+      --releasever "{{ item.release_version }}"
+      --basearch "{{ item.base_arch }}"
+  loop: "{{ satellite_repositories_minor }}"
+
+- name: "Enable repositories for major releases"
+  command: > 
+    hammer
+      -u "{{ satellite_username }}"
+      -p "{{ satellite_password }}"
+      repository-set enable 
+      --organization "{{ satellite_organization }}"
+      --product "{{ item.product }}"
+      --name "{{ item.name }}"
+      --basearch "{{ item.base_arch }}"
+  loop: "{{ satellite_repositories_major }}"
+
+- name: "Get Red Hat repo ids"
+  shell: hammer -u "{{ satellite_username }}" -p "{{ satellite_password }}" repository list --organization "{{ satellite_organization }}" | grep "Red Hat" | awk '{print $1}'
+  register: repoids
+
+- name: "Synchronize Red Hat repos"
+  command: >
+    hammer
+      -u "{{ satellite_username }}"
+      -p "{{ satellite_password }}"
+      repository synchronize
+      --id "{{ item }}"
+      --organization "{{ satellite_organization }}"
+  async: 1000
+  poll: 0
+  with_items:
+    - "{{ repoids.stdout.split('\n') }}"
+
+- name: "Create activation key"
+  command: > 
+    hammer
+      -u "{{ satellite_username }}"
+      -p "{{ satellite_password }}"
+      activation-key create
+      --name "{{ satellite_activation_key }}"
+      --lifecycle-environment Library
+      --organization "{{ satellite_organization }}"
+  
+- name: "Add subs to activation key"
+  command: >
+     hammer
+       -u "{{ satellite_username }}"
+       -p "{{ satellite_password }}"
+       activation-key add-subscription
+       --name "{{ satellite_activation_key }}"
+       --subscription-id "1"
+       --organization "{{ satellite_organization }}"
+
+- name: Ensure firewalld is not running
```

Comment:
This needs to be removed since you are not going to install it or run it.

Reply:
Done